content/post/kenobi-boot2-root-writeup.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Kenobi Boot2Root Write-Up"
+date: 2020-11-14T13:43:28-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Kenobi host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+The Kenobi boot2root was a ton of fun because it required multiple pivots to learn enough to leak a key. On my first pass i ended up overlooking the NFS shares which really impressed the importance of carefully reviewing the scans.
+
+## Environment
+The attack take place on a flat network consisting of my attack host, which is a freshly-booted Kali Linux livecd, and the the target host, a freshly-booted Linux VM that I knew contained 2 flags to capture. Nothing else was known about the host prior to the attack.
+
+## Attacking Kenobi
+Given that I knew I would be attacking a single target, I exported the host's IP as the environment variable `ATTACKTARGET`. I've also simplified a few scans for the single host case by doing things like disabling pings.
+
+### Host Enumeration
+I started the attack by opening up a new tmux session and starting a few nmap scans to determine the lay of the land, initially starting with a SYN, OS and Service Version scan.
```

Comment:
I began the attack by opening up ...

... to get a feel for the lay of the land ...

I started with a SYN, OS, and Service Version scan.